samples/getting_started/README.md

```diff
@@ -0,0 +1,47 @@
+# Getting Started with ⚡️ Bolt for Python
+> Slack app example from 📚 [Getting started with Bolt for Python][1]
```

Comment:
I added a `samples/getting_started` sample app to go along with the Getting Started guide.

Tried my best to match the patterns that @seratch used in the other sample apps including a `.gitignore` for the virtual environment and `requirements.txt` to install the dependencies. Appreciate a review to ensure it's up-to-'snuff!

Reply:
@shaydewael Would you like to link to this sample app from the _Getting Started_ guide?

Reply:
Yes! Definitely! It'd be very very helpful. I would link both at the beginning and end of the guide.

Reply:
Sounds good! Linked at the top and bottom of the guide in commit 40c0a95